### ISSUE TITLE
UnicodeDecodeError - for environment variable with non-ascii character value

### ISSUE THREAD

**Issue by ivanmoore** [docker/compose#1732]
```
$ docker-compose --version
docker-compose version: 1.3.2
CPython version: 2.7.9
OpenSSL version: OpenSSL 1.0.1e 11 Feb 2013

$ cat docker-compose.yml 
code:
  image: busybox

$ docker-compose run -e FOO=bär code ls
Traceback (most recent call last):
  File "<string>", line 3, in <module>
  File "/code/build/docker-compose/out00-PYZ.pyz/compose.cli.main", line 32, in main
  File "/code/build/docker-compose/out00-PYZ.pyz/compose.cli.docopt_command", line 21, in sys_dispatch
  File "/code/build/docker-compose/out00-PYZ.pyz/compose.cli.command", line 34, in dispatch
  File "/code/build/docker-compose/out00-PYZ.pyz/compose.cli.docopt_command", line 24, in dispatch
  File "/code/build/docker-compose/out00-PYZ.pyz/compose.cli.command", line 66, in perform_command
  File "/code/build/docker-compose/out00-PYZ.pyz/compose.cli.main", line 342, in run
  File "/code/build/docker-compose/out00-PYZ.pyz/compose.service", line 225, in create_container
  File "/code/build/docker-compose/out00-PYZ.pyz/compose.container", line 38, in create
  File "/code/build/docker-compose/out00-PYZ.pyz/docker.client", line 237, in create_container
  File "/code/build/docker-compose/out00-PYZ.pyz/docker.utils.utils", line 528, in create_container_config
UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 1: ordinal not in range(128)
```

**Comment by claudiu-cristea** [comment]
This happens also when the environment variables are loaded from a file with `env_file: ./somefile` and inside the file some environment variables have values containing non-ASCII characters.

```
$ docker-compose version
docker-compose version: 1.4.2
docker-py version: 1.3.1
CPython version: 2.7.9
OpenSSL version: OpenSSL 1.0.1j 15 Oct 2014
```

Error output:

```
Traceback (most recent call last):
  File "<string>", line 3, in <module>
  File "/compose/build/docker-compose/out00-PYZ.pyz/compose.cli.main", line 39, in main
  File "/compose/build/docker-compose/out00-PYZ.pyz/compose.cli.docopt_command", line 21, in sys_dispatch
  File "/compose/build/docker-compose/out00-PYZ.pyz/compose.cli.command", line 27, in dispatch
  File "/compose/build/docker-compose/out00-PYZ.pyz/compose.cli.docopt_command", line 24, in dispatch
  File "/compose/build/docker-compose/out00-PYZ.pyz/compose.cli.command", line 59, in perform_command
  File "/compose/build/docker-compose/out00-PYZ.pyz/compose.cli.main", line 313, in run
  File "/compose/build/docker-compose/out00-PYZ.pyz/compose.project", line 274, in up
  File "/compose/build/docker-compose/out00-PYZ.pyz/compose.service", line 377, in execute_convergence_plan
  File "/compose/build/docker-compose/out00-PYZ.pyz/compose.service", line 284, in create_container
  File "/compose/build/docker-compose/out00-PYZ.pyz/compose.container", line 42, in create
  File "/compose/build/docker-compose/out00-PYZ.pyz/docker.client", line 243, in create_container
  File "/compose/build/docker-compose/out00-PYZ.pyz/docker.utils.utils", line 534, in create_container_config
UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 3: ordinal not in range(128)
```

**Comment by dnephin** [comment]
Both of these cases are fixed in master and the 1.5.0 release. You can try it out with the release candidate:  https://github.com/docker/compose/releases/tag/1.5.0rc1

**Comment by dnephin** [comment]
Oops, I was wrong. They're only fixed on python3.
